fishtest/fishtest/templates/nn_upload.mak

```diff
@@ -0,0 +1,35 @@
+<%inherit file="base.mak"/>
+
+<header style="text-align: center; padding-top: 7px">
+  <legend>Neural Network Upload</legend>
+
+  <section class="instructions" style="margin-bottom: 35px">
+    <p>
+    Please read the
+    <a href="https://github.com/glinscott/fishtest/wiki/Creating-my-first-test">Testing Guidelines</a>
+    before uploading your network.
+    </p>
+    <p>
+    By uploading you license your network under a
+    <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0</a> or
```

Comment:
@tomtor @vondele I'm wondering how can work an "or" between two licenses.

Reply:
Yes, I wondered a bit about that too. 

Dual licensing is however possible in general. The party using the nn has to comply with one of the offered licenses.

@vondele might have suggestions for other wording.

Reply:
With a "xor" we should add a couple of checkboxes and keep trace of the user choice.

Reply:
We should pick one, it can't be an or, we wouldn't know which one. The main difference between CC0 and CC BY is that we have the attribution clause. CC0 is 'easiest' in that we have no requirements on sharing the data. If we go for CC BY we would need to be able to list the networks with their author (i.e. capture that info on upload and at least list networks with their author somewhere). Possibly best to go CC0, since if we would violate the terms of the license (i.e. not able to ack), the license would be invalid

Reply:
CC0 picked